client/src/main/java/org/apache/rocketmq/client/producer/LocalTransactionExecutor.java

```diff
@@ -18,6 +18,6 @@
 
 import org.apache.rocketmq.common.message.Message;
 
-public interface LocalTransactionExecuter {
-    public LocalTransactionState executeLocalTransactionBranch(final Message msg, final Object arg);
+public interface LocalTransactionExecutor {
```

Comment:
This is why I make these changes thorough PR process instead of regarding them trivial and merging directly.

We can discuss to figure out an acceptable way.

You may also notice that I do not change configuration name spelling issues yet and I am expecting opinions too.